File: Python/challenge_bonus.py
"""
Create variable, that is both True and False. ;)

Be creative, there are multiple correct answers. But you can't make any changes in unit tests!!! Not even rename the variable.
"""
import unittest

SCHROEDINGER = None


class SchroedingerTestCase(unittest.TestCase):
    """
    Unit tests
    """

    def test_true(self):
        """
        Should pass!
        """
        self.assertEqual(SCHROEDINGER, True)
        self.assertEqual(SCHROEDINGER, True)
        self.assertEqual(SCHROEDINGER, False)
        self.assertEqual(SCHROEDINGER, False)


if __name__ == "__main__":
    unittest.main()

File: C#/CodeFights/CodeFightsTests/Challenge_Bonus.cs
/*
Create variable, that is both True and False. ;)

Be creative, there are multiple correct answers. But you can't make any changes in unit tests!!! Not even rename the variable.
*/

namespace CodeFights
{
    using NUnit.Framework;
    using System;


    [TestFixture]
    public class Challenge_Bonus
    {        
        [Test]
        public void SchroedingerTestCase()
        {
            /*
             * Should pass!
             */
            bool SCHROEDINGER = false;
            Assert.IsTrue(SCHROEDINGER == true && SCHROEDINGER == false);

        }
    }
}